doc/examples/collision_environments/CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
-catkin_install_python(PROGRAMS
-  scripts/collision_scene_example.py
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-)
+add_executable(collision_scene_example
```

Comment:
This looks like more than a formatting change?

Reply:
Sorry, I didn't mean to commit these files, will clean up this branch